01week/resume/firstproject/css/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+
+background-color: #FOFOE1;
+color: black;
+font-family: Times;
```

Comment:
might wan to double check this value, I don't think `Times` is a valid font-family